modules/alloydb/outputs.tf

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  #  _all_instances = merge(
+  #    { primary = google_alloydb_instance.primary },
+  #    google_alloydb_instance.replicas
+  #  )
+  _all_instances = merge({ primary = google_alloydb_instance.primary }, var.cross_region_replication.enabled ? { secondary = google_alloydb_instance.secondary[0] } : {})
```

Comment:
break into multiple lines.

Also, keep the keys, and just set secondary to null when it's missing:

```hcl
_all_instances = {
  primary   = google_alloydb_instance.primary
  secondary = one(google_alloydb_instance.secondary)
}
```

Reply:
Updated!